### PR TITLE
Sean / DMT5 signal trading page

### PR DESCRIPTION
**Pull request by sean-binary**
# Description

Summary

Changes:

-   Added new DMT5 signal trading page
-   Added optional notice prop for side tab component

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by suthesan-deriv** [comment]
seems to have a conflict

File: src/components/svgs/signals.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//TODO: this a temporary way to display these svg
+import React from 'react'
```
Comment (mustofa-binary) [inline src/components/svgs/signals.js:2]: lets convert this to use img tag instead

File: src/images/svg/info2.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```
Comment (mustofa-binary) [inline src/images/svg/info2.svg:1]: please move this to the specific folder for mt5 to reduce svg population in folder 😅

File: src/pages/dmt5-trading-signals/_signal-steps.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import Tabs from '../trade-types/components/_side-tab'
+import { Show } from '../../components/containers'
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/_signal-steps.js:5]:
```suggestion
import { Show } from 'components/containers'
```

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/components/svgs/signals.js (hunk 1 above)
Comment (seanho96) [reply src/components/svgs/signals.js]: Alright

File: src/pages/dmt5-trading-signals/_subscription.js [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+import { graphql, useStaticQuery } from 'gatsby'
+import { Text, Header, QueryImage, LocalizedLinkText } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+import { Container, SectionContainer, Flex } from 'components/containers'
+import { LinkButton } from 'components/form'
+import device from 'themes/device'
+
+const query = graphql`
+    query {
+        signals: file(relativePath: { eq: "dmt5_signals.png" }) {
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/_subscription.js:12]: can we move it to its specific folder 🙏

File: src/pages/dmt5-trading-signals/_subscription.js [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+        renew: file(relativePath: { eq: "dmt5_renew.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const StyledSectionContainer = styled(SectionContainer)`
+    margin: 0;
+    padding: 0;
+    background-color: var(--color-white);
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/_subscription.js:27]: fyi, section container has props for background color, and padding, you can pass white in `background` attribute as well. but its fine too to make it in styled component

File: src/pages/dmt5-trading-signals/_subscription.js [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+    text-align: center;
+    font-size: 4.8rem;
+
+    @media ${device.tabletL} {
+        font-size: 32px;
+    }
+`
+
+const StyledText = styled(Text)`
+    font-size: 1.6rem;
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/_subscription.js:50]: by default text size is 1.6 rem

File: src/pages/dmt5-trading-signals/_subscription.js [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+    line-height: 3.6rem;
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/_subscription.js:51]: we standardize line height in text component, i dont think we need this

File: src/pages/dmt5-trading-signals/index.js [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+    height: 2px;
+    background-color: var(--color-grey-21);
+`
+
+const DMT5TradingSignals = () => {
+    const [active_tab, setTab] = useTabState()
+
+    React.useEffect(() => {
+        if (getLocationHash() === active_tab) return
+        if (getLocationHash().length === 0) {
```
Comment (mustofa-binary) [inline src/pages/dmt5-trading-signals/index.js:81]: we have useTabState hooks to handle all of this, abdalla created it, you can discuss with him for implementation 🙏

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
please help to rebase this

**Pull request merged**